### PR TITLE
Refactor data registration

### PR DESCRIPTION
**Pull request by justinwb**

### CODE REVIEW

File: proposals/specification/agent.bs [hunk 1]
```diff
@@ -7,15 +7,18 @@ Agent {#agents}
 
 An [=Agent=] is denoted by an [=identity=]. Dereferencing that [=identity=]
 leads to the [=Agent=], and a graph of useful information about them. This
-graph is used by the [=Agent=] to locate and access their own data, and data
-relevant to them, across the decentralized Web. Other [=Agents=] and
-[=Applications=] use this information to interact with them.
+graph is used by the [=Agent=] to lookup their own data, as well as data
```
Comment (TallTed) [inline proposals/specification/agent.bs:10]:
```suggestion
graph is used by the [=Agent=] to look up their own data, as well as data
```

File: proposals/specification/application-registration.bs [hunk 2]
```diff
@@ -32,15 +32,15 @@ focused elements.
   <figcaption>Data model for application registration</figcaption>
 </figure>
 
-An [=Agent=] links to an [=Application Registry Sets] via the
+An [=Agent=] links to a [=Application Registry Sets=] via the
```
Comment (TallTed) [inline proposals/specification/application-registration.bs:35]:
```suggestion
An [=Agent=] links to an [=Application Registry Sets=] via the
```

File: proposals/specification/data-registration.bs [hunk 3]
```diff
@@ -627,152 +619,144 @@ to the subsequently created resource.
...
-of access will be.
+Organizing data by [=shape tree=] creates natural permission boundaries,
+making authorization more intuitive and manageable.
 
-It is imperative that the permission model enable patterns that
-allow the [=Agent=] to make intuitive decisions, without
-having to directly manipulate [=acl resources=] for individual documents
-and containers throughout the [=Data Registry=].
+Access is granted to a certain type of data by granting access
+to the [[#datareg-all|entire Data Registration]], or to [[#datareg-specific|specific Registered Data Instances]] within it.
```
Comment (TallTed) [inline proposals/specification/data-registration.bs:748]:
```suggestion
to the [[#datareg-all|entire Data Registration]], or to
[[#datareg-specific|specific Registered Data Instances]] within it.
```

File: proposals/specification/interop.shex [hunk 4]
```diff
@@ -56,6 +56,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
   interop:registeredAt xsd:dateTime ;
   interop:updatedAt xsd:dateTime ;
   interop:registeredShapeTree IRI ;
+  interop:registeredType IRI* ;
```
Comment (joshdcollins) [inline proposals/specification/interop.shex:59]: Probably worth a discussion about whether this registeredType should be required or not.

File: proposals/specification/interop.shex [hunk 5]
```diff
@@ -56,7 +56,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
   interop:registeredAt xsd:dateTime ;
   interop:updatedAt xsd:dateTime ;
   interop:registeredShapeTree IRI ;
-  interop:hasRegisteredDataInstance IRI*
+  interop:registeredType IRI*
```
Comment (joshdcollins) [inline proposals/specification/interop.shex:59]: Should this be optional?

Comment (joshdcollins) [reply proposals/specification/interop.shex:59]: Misread, it is already optional.

**Review by joshdcollins — APPROVED** [review]

**Pull request merged**